### PR TITLE
Pick the correct 32-bit Program Files directory regardless node bitness

### PR DESCRIPTION
**Pull request by teobugslayer**
Fixes http://teampulse.telerik.com/view#item/289843

### CODE REVIEW

File: mobile/wp8/wp8-emulator-services.ts [hunk 1]
```diff
@@ -32,11 +32,15 @@ class Wp8EmulatorServices implements Mobile.IEmulatorPlatformServices {
...
 		return (() => {
 			this.$logger.info("Starting Windows Phone Emulator");
-			var emulatorStarter = path.join(process.env.ProgramFiles, Wp8EmulatorServices.WP8_LAUNCHER_PATH, Wp8EmulatorServices.WP8_LAUNCHER);
-			this.$childProcess.spawn(emulatorStarter, ["/installlaunch", app, "/targetdevice:xd"], { stdio:  ["ignore", "ignore", "ignore"], detached: true }).unref();
+			var emulatorStarter = path.join(this.programFilesPath, Wp8EmulatorServices.WP8_LAUNCHER_PATH, Wp8EmulatorServices.WP8_LAUNCHER);
+			this.$childProcess.spawn(emulatorStarter, ["/installlaunch", app, "/targetdevice:xd"], { stdio:  "ignore", detached: true }).unref();
 		}).future<void>()();
 	}
 
+	private get programFilesPath(): string {
```
Comment (ligaz) [inline mobile/wp8/wp8-emulator-services.ts]: Can this be made static?

File: mobile/wp8/wp8-emulator-services.ts [hunk 2]
```diff
@@ -32,11 +32,15 @@ class Wp8EmulatorServices implements Mobile.IEmulatorPlatformServices {
 	startEmulator(app: string, emulatorOptions?: Mobile.IEmulatorOptions) : IFuture<void> {
 		return (() => {
 			this.$logger.info("Starting Windows Phone Emulator");
-			var emulatorStarter = path.join(process.env.ProgramFiles, Wp8EmulatorServices.WP8_LAUNCHER_PATH, Wp8EmulatorServices.WP8_LAUNCHER);
-			this.$childProcess.spawn(emulatorStarter, ["/installlaunch", app, "/targetdevice:xd"], { stdio:  ["ignore", "ignore", "ignore"], detached: true }).unref();
+			var emulatorStarter = path.join(this.programFilesPath, Wp8EmulatorServices.WP8_LAUNCHER_PATH, Wp8EmulatorServices.WP8_LAUNCHER);
```
Comment (Fatme) [inline mobile/wp8/wp8-emulator-services.ts]: Is it possible to have x64 machine with x64 bit wp8 emulator?

Comment (teobugslayer) [inline mobile/wp8/wp8-emulator-services.ts]: No - at least up to VS 2015 RC

File: mobile/wp8/wp8-emulator-services.ts (hunk 1 above)
Comment (teobugslayer) [inline mobile/wp8/wp8-emulator-services.ts]: Sure!

**Comment by teobugslayer** [comment]
:+1: from @fatme

**Pull request merged**